src/transforms/hexbin.js

```diff
@@ -79,16 +79,21 @@ export function hexbin(outputs = {fill: "count"}, {binWidth, ...options} = {}) {
 
     // Construct the output channels, and populate the radius scale hint.
     const binChannels = {
-      x: {value: BX},
-      y: {value: BY},
+      x: {value: BX, source: null}, // or {value: map(BX, scales.x.invert), scale: "x"}?
+      y: {value: BY, source: null}, // or {value: map(BY, scales.y.invert), scale: "y"}?
```

Comment:
Using scales.x.invert would fail on map projections; instead, I've tried to have both Plot.tip and Plot.crosshair, but crosshair+hexbin doesn't work as I expected.  (Note that crosshair also doesn't work with map projections, independently of hexbin, but maybe that's OK.)